images/wkdev_sdk/Containerfile

```diff
@@ -81,7 +81,8 @@ ENV RUSTUP_HOME="/opt/rust" \
     CARGO_HOME="/opt/rust" \
     PATH="/opt/rust/bin:${PATH}"
 
-RUN rustup default stable && \
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
```

Comment:
You can remove `rustup` from `images/wkdev_sdk/required_system_packages/04-devtools.lst`.

For arguments we prefer the full names rather than abbreviated versions so its more readable.